Description: the concatenated jquery-ui.css in the upstream 1.12.1 release is
 slightly different from the source. Match them by adjusting the source.
Author: Paul Gevers <elbrus@debian.org>
Reviewed-By: Yadd <yadd@debian.org>
Last-Update: 2021-11-08

--- a/themes/base/theme.css
+++ b/themes/base/theme.css
@@ -437,10 +437,10 @@
 /* Overlays */
 .ui-widget-overlay {
 	background: #aaaaaa/*{bgColorOverlay}*/ /*{bgImgUrlOverlay}*/ /*{bgOverlayXPos}*/ /*{bgOverlayYPos}*/ /*{bgOverlayRepeat}*/;
-	opacity: .3/*{opacityOverlay}*/;
-	-ms-filter: "alpha(opacity=30)"/*{opacityFilterOverlay}*/; /* support: IE8 */
+	opacity: .003/*{opacityOverlay}*/;
+	-ms-filter: "alpha(opacity=.3)"/*{opacityFilterOverlay}*/; /* support: IE8 */
 }
 .ui-widget-shadow {
-	-webkit-box-shadow: 0/*{offsetLeftShadow}*/ 0/*{offsetTopShadow}*/ 5px/*{thicknessShadow}*/ #666666/*{bgColorShadow}*/;
-	box-shadow: 0/*{offsetLeftShadow}*/ 0/*{offsetTopShadow}*/ 5px/*{thicknessShadow}*/ #666666/*{bgColorShadow}*/;
+	-webkit-box-shadow: 0px/*{offsetLeftShadow}*/ 0px/*{offsetTopShadow}*/ 5px/*{thicknessShadow}*/ #666666/*{bgColorShadow}*/;
+	box-shadow: 0px/*{offsetLeftShadow}*/ 0px/*{offsetTopShadow}*/ 5px/*{thicknessShadow}*/ #666666/*{bgColorShadow}*/;
 }
